pkg/build/controller/build/podcreationstrategy.go

```diff
@@ -12,7 +12,7 @@ import (
 // buildPodCreationStrategy is used by the build controller to
 // create a build pod based on a build strategy
 type buildPodCreationStrategy interface {
-	CreateBuildPod(build *buildv1.Build, includeAdditionalCA bool) (*corev1.Pod, error)
+	CreateBuildPod(build *buildv1.Build, additionalCAs map[string]string, includeServiceCA bool) (*corev1.Pod, error)
```

Comment:
Update the interface to accept the full map of additional CAs, reused bool to indicate if we should include the service CA annotation.